console/src/view/task.rs

```diff
@@ -35,6 +35,16 @@ impl TaskView {
         let task = &*self.task.borrow();
         const DUR_PRECISION: usize = 4;
 
+        let fields: String = task
+            .fields()
+            .iter()
+            .fold(String::new(), |mut res, f| {
+                write!(&mut res, "{}={} ", f.name, f.value).unwrap();
+                res
+            })
+            .trim_end()
+            .into();
```

Comment:
it would be nice to do the same formatting here as we do in the tasks list view? but again, we can add that in a follow-up if you prefer.